package.json

```diff
@@ -4,14 +4,17 @@
   "scripts": {
     "codegen": "graph codegen",
     "build": "graph build",
-    "auth": "graph auth https://api.thegraph.com/deploy/ $GRAPH_ACCESS_TOKEN",
-    "deploy": "graph deploy dan13ram/seed-graph --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/ --debug",
+    "oldauth": "graph auth https://api.thegraph.com/deploy/ $GRAPH_ACCESS_TOKEN",
```

Comment:
Any reason why you kept this? Don't hesitate to delete things that are no longer used; we have things like source control to unearth historical code if need be.